init-container/main.go

```diff
@@ -33,6 +41,7 @@ func main() {
             secrets = append(secrets, Secret{
                 Id: secretArn,
                 Region: parsedArn.Region,
+                ExplodeJson: envExplodeJsonKeys,
```

Comment:
Please add this to the 'envSecretNames' block as well

Reply:
Good catch 👍